### ISSUE TITLE
Revert #825: `colors` terminal DOS

### ISSUE THREAD

**Issue by Rauno56** [open-telemetry/opentelemetry-js-contrib#826]
To avoid people running into problems with broken `colors` package, I added `colors@1.4.0` to `package.json`, that needs to be reverted when the situation with the `colors` package is resolved.

+ one additional one from [another PR](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/821/files#diff-90eecd12c99a9d96beea5af4ce3eba598977225301cbed2aa59302c34bddaedfR61)

**Comment by dyladan** [comment]
Tracking issue in colorsjs repo: https://github.com/Marak/colors.js/issues/285

**Comment by vmarchaud** [comment]
After reading a bunch on reddit it seems that the author will not fix the package anytime soon

**Comment by dyladan** [comment]
Could be also that karma removes the dependency on colors

**Comment by dyladan** [comment]
They may also switch to the fork by one of the colors maintainers https://github.com/DABH/colors.js

**Comment by Rauno56** [comment]
> Tracking issue in colorsjs repo: [Marak/colors.js#285](https://github.com/Marak/colors.js/issues/285)

That tracking issue is a joke. It's the malicious activity by the maintainer. We depend on the `colors` through Karma - they need to fix before we can act on this issue.

**Comment by Flarna** [comment]
Seems npm removed colors >1.4.0 meanwhile. Not sure if this should be the trigger to revert.
